pkg/restore/restore.go

```diff
@@ -733,18 +733,59 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 		createdObj, restoreErr := resourceClient.Create(obj)
 		if apierrors.IsAlreadyExists(restoreErr) {
 			equal := false
```

Comment:
you can remove this and declare it inline on L755 (`equal := equality.Semantic.DeepEqual(...)`)

Reply:
Actually, you can avoid declaring it altogether and just change L757 to be `if !equality.Semantic.DeepEqual(...) {`